assets/src/edit-story/panels/position.js

```diff
@@ -28,22 +29,22 @@ function PositionPanel( { selectedElements, onSetProperties } ) {
 	return (
 		<Panel onSubmit={ handleSubmit }>
 			<Title>
-				{ 'Position' }
+				{ __( 'Position', 'amp' ) }
 			</Title>
 			<InputGroup
-				label="X"
+				label={ __( 'X', 'amp' ) }
```

Comment:
Let's use `_x()` to explain what "X", "Y", and "px" mean here.

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/4143/commits/8e29fd2233e19ebfeb7f1d1dfdda23ca3dc8fc19